docs/azure/partials/common/_dns-resource-group.mdx

```diff
@@ -1 +1 @@
-The `dns-azure-resource-group` is only required if using Azure as your DNS provider. This is the name of the resource group where the [DNS Zone](https://learn.microsoft.com/en-us/azure/dns/dns-zones-records) resource is provisioned.
+The `dns-azure-resource-group` flag is only used Azure is your DNS provider. This is the name of the resource group where the [DNS Zone](https://learn.microsoft.com/en-us/azure/dns/dns-zones-records) resource is provisioned. This flag is not required, but may be useful if you have multiple DNS Zones with the same URL configured. If the resource group is not provided, the first DNS Zone that matches the given URL in your subscription will be used.
```

Comment:
Update that first bit to "The `dns-azure-resource-group` flag is only used if Azure is your DNS provider..." and we're good to go